README.md

```diff
@@ -11,23 +11,11 @@ the host's storage, memory and networks as well as virtual machine
 creation, other host administration tasks, statistics gathering, and
 log collection.
 
-
-This repository is hosted on [gerrit.ovirt.org:vdsm](https://gerrit.ovirt.org/#/admin/projects/vdsm)
-and a **backup** of it is hosted on [GitHub:vdsm](https://github.com/oVirt/vdsm)
```

Comment:
Why not keeping an info here about the official repository location and about the old gerrit repository for history references?

Reply:
Good idea, I'll add a link to the old repository for looking up old review comments.

Reply:
I forgot to add some non storage tests that I own, will update the list on the next push.